packages/rn-tester-e2e/package.json

```diff
@@ -14,9 +14,6 @@
     "test-e2e": "node ./../../scripts/e2e/run-e2e-tests.js"
   },
   "devDependencies": {
-    "appium": "^2.0.0",
-    "appium-uiautomator2-driver": "^2.29.1",
-    "appium-xcuitest-driver": "^4.32.10",
```

Comment:
are you sure we don't need these when using it locally? I'm pretty sure you need them

Reply:
Let's see how will this work in CI, locally tests are passing correctly

Reply:
is that because in the readme we tell people to install the tooling globally on their machine?